docs/main/prerequesites.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Prerequisites
+description: Operating system and network requirements
+last_updated: Fri September 20 2019
```

Comment:
There is a danger of this last updated time going out of date. Is there a way to use Git for this?

Reply:
I don't think it will go out of date, if you edit the file you edit the front matter line.